lib/attr_encrypted/adapters/active_record.rb

```diff
@@ -43,32 +50,47 @@ def attributes_with_attr_encrypted=(*args)
           end
         end
 
+        # Saves the attr class wrappers, so AR can infer the correct type later
+        def encrypted_reflections
```

Comment:
Are the reflections encrypted? I don't believe they are. If so, this name is misleading.